modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenProperty.java

```diff
@@ -54,6 +54,8 @@ public class CodegenProperty implements Cloneable, IJsonSchemaValidationProperti
     @Getter @Setter
     public String datatypeWithEnum;
     @Getter @Setter
+    public String rawDatatypeWithEnum;
```

Comment:
Instead of introducing a new field here, could you please define a new lambda (e.g. "removeAnnotations") in `AbstractJavaCodegen`? Similar to this: https://github.com/OpenAPITools/openapi-generator/blob/d29196a1f0ad41fc17f5d4875650f87480d66a40/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java#L653 You should then be able to use it like this in the mustache templates: `{{#removeAnnotations}}{{{datatypeWithEnum}}}{{/removeAnnotations}}`.
Using a lambda instead of a field eliminates the risk that future contributors make changes to `datatypeWithEnum` without keeping `rawDatatypeWithEnum` in sync.